courseaffils/tests/urls.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import unicode_literals
 
-from django.conf.urls import url
+from django.urls import re_path
 from courseaffils import views
 
 
 urlpatterns = [
-    url(r'^select_course/$',
-        views.CourseListView.as_view(),
-        name='select_course'),
-    url(r'^course/create/$',
-        views.CourseCreateView.as_view(),
-        name='create_course'),
+    re_path(r'^select_course/$',
```

Comment:
Because these views don't appear to be using any regex-specific stuff, just using the more standard `path` should work as well:
```
path('select_course/',
    views.CourseListView.as_view(),
    name='select_course'),
```